starter-project-web-react/types/Blog.ts

```diff
@@ -0,0 +1,36 @@
+export interface Blog {
+        _id: "",
+        authorUserId: {
+          fullName: "",
+          email: "",
+          __v: 0
+        },
+        title: "",
+        content: "",
+        imageUrls: [],
+        rating: {},
+        description: "",
+        createdAt: "",
+        updatedAt: "",
+        __v: 0
+      
+}
+
+export interface Comment {
```

Comment:
The interfaces should hold only the type declaration, not a default value you can create another object for storing default value